packages/grafana-data/src/types/featureToggles.gen.ts

```diff
@@ -36,4 +36,5 @@ export interface FeatureToggles {
   newNavigation?: boolean;
   showFeatureFlagsInUI?: boolean;
   disable_http_request_histogram?: boolean;
+  ['validated-queries']?: boolean;
```

Comment:
although it is not required...  it would be great to see everything use camelCase.